spec/rubocop/config_loader_spec.rb

```diff
@@ -386,6 +387,14 @@
       )
     end
 
+    it 'loads configuration properly when it includes non-ascii characters ' do
+      create_file(configuration_path, ['# All these cops of mine are ❤',
```

Comment:
Not sure if I completely understand the results you're getting. Could you post the ouput of `bundle exec ascii_spec` and `bundle exec ruby -Ispec -rrspec/autorun -E ascii ./spec/rubocop/config_loader_spec.rb`, both _with_ and _without_ the following patch?

```diff
diff --git a/lib/rubocop/config_loader.rb b/lib/rubocop/config_loader.rb
index ec2a11e..abc4248 100644
--- a/lib/rubocop/config_loader.rb
+++ b/lib/rubocop/config_loader.rb
@@ -139,7 +139,7 @@ module RuboCop
       end
 
       def load_yaml_configuration(absolute_path)
-        yaml_code = IO.read(absolute_path)
+        yaml_code = IO.read(absolute_path, encoding: 'UTF-8')
         # At one time, there was a problem with the psych YAML engine under
         # Ruby 1.9.3. YAML.load_file would crash when reading empty .yml files
         # or files that only contained comments and blank lines. This problem
```